app/controllers/ballots_controller.rb

```diff
@@ -47,7 +52,7 @@ def load_ballot
 
   def load_all_beers
     @beers =
-      if brewery = Poll.current.brewery.presence
+      if brewery = Poll.current.try(:brewery)
```

Comment:
Yeah the issue is it crashing when there is no poll. When there's no poll you're calling brewery on nil.